addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -883,6 +883,11 @@ body.editor_enable.editor_has_snippets {
             }
         }
 
+        // Buttons with an icon but no text, so no possible overflow
+        we-button.o_we_button_icon {
+            flex: 0 0 auto;
+        }
```

Comment:
There should be a dedicated commit about this... or is it the first button with an icon that we introduce in the panel? :thinking: I doubt it (and if not the first button, those buttons should be fixed, probably in stable versions). As I explained to @rdeodoo I also would like the developer/designer to not have to think about adding this class:
- Either it's a button with a fa icon: it should be define with `<we-button class="... fa fa-machin ..."` -> the CSS rule can detect `we-button.fa` (as I thought it was already the case at some point)
- Or it's a button with a image icon -> it is defined with `<we-button data-img="..."` -> the JS which sets up that image can also automatically add the `o_we_button_icon` class if necessary